pkgs/applications/misc/kitty/default.nix

```diff
@@ -1,19 +1,19 @@
-{ stdenv, fetchFromGitHub, pkgs, python3Packages, glfw, libunistring, glew, fontconfig, zlib, pkgconfig, ncurses, imagemagick, makeWrapper }:
+{ stdenv, fetchFromGitHub, pkgs, python3Packages, glfw, libunistring, harfbuzz, fontconfig, zlib, pkgconfig, ncurses, imagemagick, makeWrapper, xsel, libstartup_notification }:
```

Comment:
You do not seem to be using `xsel` anywhere. You will want to replace it [here](https://github.com/kovidgoyal/kitty/blob/8cad6ae9d6a26efcdc26f9d9338cb034a9027e32/kitty/utils.py#L180) with the full path or add it to PATH in the `installPhase`.

Same for `libstartup_notification`, replace `find_library` in https://github.com/kovidgoyal/kitty/blob/8cad6ae9d6a26efcdc26f9d9338cb034a9027e32/kitty/utils.py#L240 with `LoadLibrary('${libstartup_notification}/lib/libstartup-notification-1.so')` or something.

Reply:
Hi, can you guide me please. I'm new to nixos so it is hard to me to understand what is problem and how to fix it.
xsel: I added it there since kitty needs it. What is problem? Why to replace it with the full path and what does it solve?
libstartup_notification: What is wrong with find_library? How to replace it? With a patch or something provided by nix?
Thanks!

Reply:
Thanks for review!